### ISSUE TITLE
Technical grid and card update for the Component page

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ods-ios#229]
#  Component page update

## General description
The component page should be updated to use:
* the ODS grid modules
* the ODS small card component

This task should be taken once the small card has been clarified

**Comment by Tayebsed93** [comment]
**Text magnification** : We recommend not to put a fixed size in the "grid card" because the text is truncated once the text is enlarged
**Voice Over** : Also be careful not to vocalize the image with Voice Over

**Comment by Tayebsed93** [comment]
bug accessibility ticket **text magnification** : https://github.com/Orange-OpenSource/ods-ios/issues/299
